website_sale_google_shopping/static/description/index.html

```diff
@@ -0,0 +1,210 @@
+<p><img src="https://img.shields.io/badge/licence-AGPL--3-blue.svg"></p>
```

Comment:
agree: too difficult to synchronize overtime.
We only need to maintain README.rst